app/assets/stylesheets/finder_frontend.scss

```diff
@@ -446,3 +446,9 @@
     }
   }
 }
+
+/* Override class to bleed more into govuk-form-group class */
+.govuk-skip-link:focus {
+  position: relative !important;
```

Comment:
I assume there's a good reason for the `!important` but we should try to avoid this - currently this rule will apply to any `govuk-skip-link` in finder-frontend.

Reply:
I agree, the reason for using `!important` is that govuk frontend uses it and so we have to override it. Anyway, I've found a better way which doesn't need this override because the only reason I added this was to try add some spacing between the skip link and "Show brexit" checkbox.